mod-source-record-manager-server/src/main/java/org/folio/services/journal/InvoiceUtil.java

```diff
@@ -52,13 +56,16 @@ public class InvoiceUtil {
   private static final String NO_INVOICE_TITLE_MESSAGE = "No content";
   private static final String NO_VENDOR_INVOICE_NUMBER = "0";
 
-  public static LinkedList<JournalRecord> buildJournalRecordByEvent(DataImportEventPayload event)
+  public static List<JournalRecord> buildJournalRecordByEvent(DataImportEventPayload event)
```

Comment:
Just for my knowledge, what is the thinking behind this change?

Reply:
I used List<JournalRecord> buildErrorJournalRecord() which returns "List" at 67 line and I think It`s improved code by SOLID principles